docs/getting-started/quickstart/README.md

```diff
@@ -0,0 +1,211 @@
+- Read the appended event with sample code using the EventStoreDB client.
+
+## Prerequisites
+
+Before starting, ensure you have the following:
+
+- A GitHub account to use GitHub Codespaces.
+- Basic knowledge of one of the development languages/platforms below.
+- Familiarity with command-line operations.
+
+## Step 1: Set up Your Codespace
+
+1. Choose one of development languages/plaforms below and click the Codespaces link:
+
+::: tabs#dev-language-platform
+
+@tab Select >
+
+
+@tab Python
+|Language/Platform|Link to Codespaces|
+|-----|--------|
+|<div style="display: flex; justify-content: center; align-items: center;"><img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/python/python-original-wordmark.svg" alt="Python" style="height: 64px;" /></div>|[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/EventStore/EventStoreDB-From-Scratch-Python?ref=stephen-getting-started&devcontainer_path=.devcontainer%2Fquickstart%2Fdevcontainer.json)|
+
+@tab Java
+|Language/Platform|Link to Codespaces|
+|-----|--------|
+|<div style="display: flex; justify-content: center; align-items: center;"><img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/java/java-original-wordmark.svg" alt="Python" style="height: 64px;" /></div>|[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/EventStore/EventStoreDB-From-Scratch-Java?ref=stephen-getting-started&devcontainer_path=.devcontainer%2Fquickstart%2Fdevcontainer.json)|
+
+@tab .NET
+|Language/Platform|Link to Codespaces|
+|-----|--------|
+|<div style="display: flex; justify-content: center; align-items: center;"><img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/dot-net/dot-net-plain-wordmark.svg" alt="Python" style="height: 64px;" /></div>|[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/EventStore/EventStoreDB-From-Scratch-.Net?ref=stephen-getting-started&devcontainer_path=.devcontainer%2Fquickstart%2Fdevcontainer.json)|
+
+@tab node.js
+|Language/Platform|Link to Codespaces|
+|-----|--------|
+|<div style="display: flex; justify-content: center; align-items: center;"><img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/nodejs/nodejs-plain-wordmark.svg" alt="Python" style="height: 64px;" /></div>|[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/EventStore/EventStoreDB-From-Scratch-Node.js?ref=stephen-getting-started&devcontainer_path=.devcontainer%2Fquickstart%2Fdevcontainer.json)|
+
+:::
+
+2. Follow the instructions to create a new Codespace.
+
+3. Wait for your Codespace to build. This will take a few moments and you will see a welcome message in the terminal.
+
+## Step 2: Start the EventStoreDB Server
+
+1. Once your Codespace is loaded, run the following custom script in the terminal to start the EventStoreDB server:
+
+   ```sh
+   ./start-cluster.sh
```

Comment:
should reallry be called `start_esdb.sh` since  it's not  a cluster that is started .
( and also avoid `start_node` to avoid having to explain node / cluster / ... )

Reply:
Created a task for this. There's some work to update other documentations that references this name, but it's not a big deal. Will work on this later 

https://eventstore-engineering.atlassian.net/jira/software/projects/ES/list?selectedIssue=ES-77